ui/src/scenes/Home/components/ResponsibilitiesContainer/ResponsibilitiesContainer.js

```diff
@@ -3,7 +3,6 @@ import PropTypes from 'prop-types';
 import gql from 'graphql-tag';
 import { withRouter } from 'react-router-dom';
 import { Query } from 'react-apollo';
-import { GET_NEED_RESPONSIBILITIES } from '@/services/queries';
```

Comment:
Same comment here about shared queries as in NeedsContainer. We use the shared query so that CreateResponsibility can use it to update the Apollo cache. Feel free to update the shared query.

Reply:
Same thing her. A warning pops up when you create a responsibility about the realizer field missing.